src/cascadia/LocalTests_SettingsModel/TerminalSettingsTests.cpp

```diff
@@ -85,7 +85,8 @@ namespace SettingsModelLocalTests
         for (int i = 0; i < expectedArgc; ++i)
         {
             const bool useQuotes = static_cast<bool>(rng(2));
-            const auto count = static_cast<size_t>(rng(64));
+            // We need to ensure there is at least one character
+            const auto count = static_cast<size_t>(rng(64) + 1);

```

Comment:
I believe this comment (and change) isn't correct, because `CommandLineToArgvW` should work even if you pass it an empty string.

Reply:
This test errored out for me whenever it generated a command line that had a non-quoted empty argument.

In particular at line 115 - `VERIFY_ARE_EQUAL(expectedArgc, argc);` 

`expectedArgc` would be one more than `argc`.

Reply:
e.g. a generated string of `aaaa  bbbb` has an `expectedArgc` of 3, but `CommandLineToArgvW()` counts 2.